apps/cli/src/models/audience-dto.ts

```diff
@@ -0,0 +1,12 @@
+import type { Static } from '@sinclair/typebox'
+import { Type } from '@sinclair/typebox'
+
+export const AudienceDto = Type.Object({
```

Comment:
We shouldn't duplicate all these types. Can either move them into a @featureboard/api-contracts lib, or reference from management api and configure the build to bundle them. First is preferable

Reply:
If I pull them out into a new library how do I get the published and reference it?

Reply:
Relying on any as types are mostly in the ejs files any way where there is no type safty